docs/wiki/feature/maptools.md

```diff
@@ -21,7 +21,10 @@ This adds the possibility to draw accurate lines on the map screen.
 ### 1.2 Map tools
 This adds map tools that can be used to measure distances between two points or bearings on the map.
 
-### 1.3 GPS on map
+### 1.3 Plotting Board
+This adds a plotting board that can be used to 
```

Comment:
I knew I had forgotten to finish something in the documentation, but couldn't remember where exactly..

Reply:
'This adds a plotting board that can be used to aid in the rapid employment and adjustment of short-ranged indirect fires, as well as quick measurements of directions and distances between points, and general land-navigation.'

Reply:
I pushed your suggestion, albeit with 2 minor modifications (`employment` -> `usage`, `fires` -> `fire`). If you disagree with these, let know.

Reply:
First is fine, second one is... strange to me. 'Indirect Fire' is like... 'air support'. It's a concept in abstract. 'Indirect fires' refers to fire missions themselves or collective assets that create them.

This is admittedly a nitpick, but it's more 'rapid usage and adjustments of [shooting in an arc]' vs 'rapid usage and adjustments of [plotted fire missions]'.

Decision is yours since neither is technically wrong, and I don't want to seem like I'm getting bent out of shape over a single letter.